komga-webui/src/router.ts

```diff
@@ -123,6 +123,7 @@ const router = new Router({
 })
 
 router.beforeEach((to, from, next) => {
+  document.title = (to.meta && to.meta.title && to.meta.title(to)) || 'Komga'
```

Comment:
the `meta.title` field is never set, so not sure what this will do exactly

Reply:
I realize I forgot to remove this. Initially I attempted to set the title to the book at this point in a per-route meta function, this is also why Route was imported.